nodejs/webrtc-hello-again/.env.default

```diff
@@ -16,4 +16,3 @@ FROM_NUMBER=
 
 # Don't change anything below here
 BANDWIDTH_WEBRTC_CALL_CONTROL_URL=https://api.webrtc.bandwidth.com/v1
-BANDWIDTH_WEBRTC_SIPX_PHONE_NUMBER=+19192892727
```

Comment:
Did it get deleted then added back?

Reply:
No I forgot to remove it from this file when I updated the sample to use SIP URI transfers instead of the magic number